.github/workflows/test.yml

```diff
@@ -45,4 +49,5 @@ jobs:
           poetry run netexec mssql 127.0.0.1
           poetry run netexec ssh 127.0.0.1
           poetry run netexec ftp 127.0.0.1
-          poetry run netexec smb 127.0.0.1 -M veeam
\ No newline at end of file
+          netexec smb 127.0.0.1 -M veeam
```

Comment:
Why add in a non-poetry run? I assume this is a mistake

Reply:
At the moment pipx does not support the lock files and therefore don't fully uses the versions specified, but resolves dependencies using the pyproject.toml. This is most of the time nearly the same but in cases like #252 pipx can have slightly different dependencies so imo it's not bad to also try the pipx installation.

Although I think we should have the repository already local due to the git runner setup. We should check if it's possible to install the local version and also cache the dependencies to minimise the runner runtime.

Reply:
> Why add in a non-poetry run? I assume this is a mistake

Mistake i will fix it